crypto/bio/socket_helper.c

```diff
@@ -20,6 +20,7 @@
 
 #if !defined(OPENSSL_TRUSTY)
 
+#include <time.h>
```

Comment:
Build in internal system does not need this, right?

Reply:
Yeah, I was confused as to why this wasn't needed in the internal system. But it's been brought up as a past issue for BoringSSL in other similar platforms, so I thought the change would be worth pushing forward.